bokehjs/src/coffee/core/util/serialization.coffee

```diff
@@ -0,0 +1,90 @@
+import * as _ from "underscore"
+
+ARRAY_TYPES =
+  float32: Float32Array
+  float64: Float64Array
+  uint8: Uint8Array
+  int8: Int8Array
+  uint16: Uint16Array
+  int16: Int16Array
+  uint32: Uint32Array
+  int32: Int32Array
+
+DTYPES = {}
+for k, v of ARRAY_TYPES
+    DTYPES[v.name] = k
+
+_arrayBufferToBase64 = (buffer) ->
+  bytes = new Uint8Array( buffer )
+  binary = (String.fromCharCode(b) for b in bytes)
+  return btoa( binary.join("") )
```

Comment:
@philippjfr, @bryevdv, `btoa` isn't imported imported. Is this code used at all? If this is used only in tests (as `global.btoa = require('btoa')` indicates), then this code should be moved to `bokehjs/test/utils.coffee`).